ui/src/containers/PleaseWaitDialog.js

```diff
@@ -7,105 +7,52 @@ import { setLoading } from '../actions/general';
 export class PleaseWaitDialog extends React.Component {
```

Comment:
I'm simplifying things a big in this component. Indirectly related to ESLint (i.e. I didn't have to go this far to fix the warning in this file).